packages/frontend-shared/src/locales/en-US.json

```diff
@@ -119,6 +119,7 @@
     "pageTitle": "Specs",
     "newSpecButton": "New Spec",
     "searchPlaceholder": "Search Specs",
+    "clearSearch": "Clear search pattern",
```

Comment:
"search field" or "search input" sounds more natural to me than "search pattern". Maybe I've been looking at code too much xD

Reply:
You're not wrong, I struggled a bit with the wording here. I agree that "Search Field" makes more sense